Dockerfile.child_chain

```diff
@@ -0,0 +1,67 @@
+FROM alpine:3.8
+
+LABEL maintainer="OmiseGO Team <omg@omise.co>"
+LABEL description="Official image for OmiseGO (Watcher) Plasma Network"
+
+ENV LANG=C.UTF-8
+
+## S6
+##
+
+ENV S6_VERSION="1.21.4.0"
+
+RUN set -xe \
+ && apk add --update --no-cache --virtual .fetch-deps \
+        curl \
+        ca-certificates \
+ && S6_DOWNLOAD_URL="https://github.com/just-containers/s6-overlay/releases/download/v${S6_VERSION}/s6-overlay-amd64.tar.gz" \
+ && S6_DOWNLOAD_SHA256="e903f138dea67e75afc0f61e79eba529212b311dc83accc1e18a449d58a2b10c" \
+ && curl -fsL -o s6-overlay.tar.gz "${S6_DOWNLOAD_URL}" \
+ && echo "${S6_DOWNLOAD_SHA256}  s6-overlay.tar.gz" |sha256sum -c - \
+ && tar -xzC / -f s6-overlay.tar.gz \
+ && rm s6-overlay.tar.gz \
+ && apk del .fetch-deps
+
+## Application
+##
+
+RUN apk add --update --no-cache --virtual .child_chain-runtime \
+        bash \
+        libressl \
+        libressl-dev \
+        lksctp-tools
+
+COPY rootfs /
+
+# USER directive is not being used here since privileges are dropped via
+# s6-setuigid in /entrypoint. s6-overlay is required to be run as root.
+ARG user=child_chain
```

Comment:
Are we rolling with `child_chain`? I know it's not related to this PR but there's a lot of inconsistency in this project between Childchain/child_chain/Child Chain etc

Reply:
I don't know what to do! :D

Reply:
There was a discussion and resulted to `child chain` is a correct form. So in this context with underscore.

Reply:
> lot of inconsistency in this project between Childchain/child_chain/Child Chain

I'd say we're pretty consistent with ChildChain/child_chain/Child chain. Two words all of these. I just searched: `Childchain` is very rare, `Rootchain` doesn't show up, there's several `childchains` and `rootchains`. We can groom this at some point.

Reply:
I agree. I don't think I've made a naming mistake in this PR. Everything is `child_chain`!